### DISCUSSION TITLE
can't show my email like others

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36705] [Other Feature Feedback, Questions, & Ideas] [answered]
hey i can't show my email in my profile is there any way?

**Comment by radar-blip** [comment]
Under your profile picture, where it says 'Edit Profile'.

**Reply by ghost** [reply]
i have tried that.

**Comment by johnmichaelarc** [comment]
Go to your profile and click "Edit Profile". and remove the email. Or go to settings privacy

**Reply by ghost** [reply]
i have also tried this.

**Comment by shirinapr** [comment, marked as answer]
1) verify your email
2) go to [settings](https://github.com/settings/emails) read under and uncheck "Keep my email addresses private".
3) add your github email in your git config.
* keep that in mind only users can see your email (ppl who are logged in github.)

**Reply by ghost** [reply]
oh it seems like i found the solution, thanks.

**Reply by shirinapr** [reply]
you're welcome. ;)

**Comment by ossanna16** [comment]
Hey there @ShilasCommiting and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by CarlosCaruci** [comment]
You must go to settings, it's very easy, much easier than learning JS I assure you.
